x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/DownsampleStepTests.java

```diff
@@ -26,6 +26,7 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
```

Comment:
perhaps java.time.temporal.ChronoUnit would be better ?

Reply:
actually it is already being used in ESTestCase so would make the PR far too big.

Reply:
Yeah that'd be quite a big change, `ChronoUnit` is a lot more general than `TimeUnit` but we wouldn't want to add support for all those other units in our APIs today.